### DISCUSSION TITLE
smtp issue

### DISCUSSION THREAD

**Discussion by salvatore-web** [Ylianst/MeshCentral#6471] [Issues] [answered]
I have MeshCentral instance on my synology NAS as container working fine, I can login with my user name and password with 2FA token.
I enabled smpt in my config.json file and restated the container.
Now I am NOT able to login, because it prompts me to email verification that I never receive.
Please Help!

**Comment by si458** [comment]
Have u checked the output in the console if smtp was successful sending the email?

You can set `"debug": "smtp"` under settings in config.json, restart the container and watch the output

**Reply by salvatore-web** [reply]
I am not to familiar with the step you described, but here is the log:

Installing modules [ 'semver@7.5.4', 'nodemailer@6.9.8', 'otplib@10.2.3' ]
MeshCentral HTTP redirection server running on port 8080.
MeshCentral v1.1.32, WAN mode, Production mode.
MeshCentral Intel(R) AMT server running on sdscomp.synology.me:4433.
MeshCentral HTTPS server running on sdscomp.synology.me:2087, alias port 2088.
Loaded web certificate from "https://sdscomp.synology.me:2088", host: "sdscomp.synology.me"
  SHA384 cert hash: 47ca880514fafdeac1581b9c5b47b4c552a1dbb544738d29d122cb75bbf15d1f61f0ecf63a0477d47a9cd7f4186e342c
SMTP mail server mail.sdscomp.net working as expected.

**Reply by si458** [reply]
Inside your config.json, you have `"settings"`
Inside there add `"debug": "smtp"`
Then restart meshcentral container and it should output some logs in the container when it tries sending a verify email

**Reply by salvatore-web** [reply]
I added "debug": "smtp" to my config.json and restarted the container, but there is nothing in the log

**Reply by si458** [reply]
Sorry my mistake change `smtp` to `email` then restart container

**Reply by salvatore-web** [reply]
OK, i get this error:

SMTP mail server mail.sdscomp.net working as expected.
EMAIL: Sending email verification to salvatore@sdscomp.net
EMAIL: Getting mail template for: account-check, lang: en-us,en
EMAIL: Error: Failed to get mail template.

**Reply by si458** [reply]
Oh dear that's not good!
What docker image are you using?
Are u using the official one `ghcr.io/ylianst/meshcentral` ?

**Reply by salvatore-web** [reply, marked as answer]
Sorry, I forgot to copy the emails folder to my meshcentral-web for customization.
everything is working fine now, i got the email verification.
Thank you for pointing me to the right direction.

**Reply by si458** [reply]
@salvatore-web oh wow glad u spotted that!
In theory it should fall back to the default email templates if they don't exist in your meshcentral-web folder!
Might need to look into that as that's a bug !?

**Reply by si458** [reply]
ok after checking it does indeed fall back but you might have a custom `webemailpaths` specified
you can check where it loads things from by running `showpaths` from the `my server console` tab
